website/api/zkvm/precompiles.md

```diff
@@ -33,39 +34,59 @@ git = "https://github.com/risc0/RustCrypto-hashes"
+```toml
+cargo update -p sha2 --precise 0.10.6
+```
+
 When using cryptography indirectly, e.g. via the `cookie`, `oauth2`, or `revm`,
 crates it may be possible to enable acceleration support without code changes by
 applying a [Cargo patch][cargo-patch].
 
+Several of our precompiles are still undergoing revision and
+review, and so users must opt-in to these features by setting the `"unstable"`
+feature flag on the `risc0-zkvm` crate used by the zkVM guest and by the
+`risc0-build` crate used to build the guest. For users who need a stable,
+production-ready version we are working on stablizing these precompiles as soon
+as possible.
+
 An example of how to use these crates to accelerate ECDSA signature verification
 can be in the [ECDSA example][ecdsa]. Note the [use of the patched
 versions][ecdsa-patched] of `sha2`, `crypto-bigint` and `k256` crates used in
 the guest's `Cargo.toml`.
 
-## Adding Accelerator Support To Crates
+## Adding Precompile Support To Crates
 
-It's possible to add accelerator support for your own crates.
+It's possible to add precompile support for your own crates.
 
 An example of how to do this can be found in this [diff of RISC Zero's k256
 crate fork][k256-diff], which shows the code changes needed to accelerate
 RustCrypto's secp256k1 ECDSA library. This fork starts from the base
 implementation, and changes the core operations to use the accelerated 256-bit
-modular multiplication instruction. E.g. [`FieldElement8x32R0::mul`][field-mul].
+elliptic curve instructions. E.g. [`lincomb`][lincomb].
 
 [^1]: This is similar to the cryptography support such as [AES-NI] or the [SHA
     extensions] for x86 processors. In both cases, the circuitry is extended to
     compute otherwise expensive operations in fewer instruction cycles.
 
 [AES-NI]: https://en.wikipedia.org/wiki/AES_instruction_set#x86_architecture_processors
-[bigint]: https://github.com/risc0/risc0/pull/466
 [cargo-patch]: https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-patch-section
 [curve25519-dalek]: https://github.com/risc0/curve25519-dalek/tree/risczero
-[ecdsa]: https://github.com/risc0/risc0/tree/main/examples/ecdsa
-[ecdsa-patched]: https://github.com/risc0/risc0/blob/main/examples/ecdsa/methods/guest/Cargo.toml#L13-L18
-[field-mul]: https://github.com/risc0/RustCrypto-elliptic-curves/compare/k256/v0.13.1..k256/v0.13.1-risczero.1#diff-ab10e01be1d99a874f90c9a6143bb1c64f37e04dcb220b5ab50b9273d99e0a0cR176-R179
+[ecdsa]: https://github.com/risc0/risc0/tree/release-1.2/examples/ecdsa
+[ecdsa-patched]: https://github.com/risc0/risc0/blob/release-1.2/examples/ecdsa/methods/guest/Cargo.toml#L13-L18
 [git-dep]: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories
 [k256-diff]: https://github.com/risc0/RustCrypto-elliptic-curves/compare/k256/v0.13.1..k256/v0.13.1-risczero.1
+[lincomb]: TODO
```

Comment:
Is this a link you want? Or did you have something else in mind?

https://github.com/risc0/RustCrypto-elliptic-curves/blob/k256/v0.13.3-risczero.1/k256/src/arithmetic/mul.rs#L349-L377